integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -156,4 +156,46 @@ public void testSearchTypeParamWithFileSpecified() throws Exception {
             assertValuePresent(metadata, 1, "type", "File");
         }
     }
+
+    @Test
+    public void testSearchWithNormalSorting() throws Exception {
+        createDefaultObjects();
+        collectionFactory.createCollection(adminUnit1,
+                Map.of("title", "A first collection", "readGroup", "everyone"));
+
+        // rollup=false disables the FileObjects getting grouped into their associated Works
```

Comment:
This comment looks like its leftover from a previous test. Same in the other test

Reply:
removed!